spring-security-modules/spring-security-web-boot-2/src/main/java/com/baeldung/multiplelogin/MultipleLoginSecurityConfig.java

```diff
@@ -44,30 +47,25 @@ public UserDetailsService userDetailsServiceApp1() {
         }
 
         @Bean
-        public SecurityFilterChain filterChainApp1(HttpSecurity http) throws Exception {
-            http.antMatcher("/admin*")
-                .authorizeRequests()
-                .anyRequest()
-                .hasRole("ADMIN")
+        public SecurityFilterChain filterChainApp1(HttpSecurity http, HandlerMappingIntrospector introspector) throws Exception {
+            MvcRequestMatcher.Builder mvcMatcherBuilder = new MvcRequestMatcher.Builder(introspector);
+            http.securityMatcher("/admin*")
+                .authorizeHttpRequests(authorizationManagerRequestMatcherRegistry ->
+                            authorizationManagerRequestMatcherRegistry.requestMatchers(mvcMatcherBuilder.pattern("/admin*")).hasRole("ADMIN"))
```

Comment:
Please remove the extra empty line and take note of the line formatting.

Reply:
Done